readme.md

```diff
@@ -771,3 +945,8 @@ Any errors that occur on any of the services will throw a StripeException with t
 
 The StripeException contains and HttpStatusCode and a StripeError entity. The StripeError entity contains the type, message, code and param. For more infomation, review the Errors section
 of stripe here: https://stripe.com/docs/api#errors
+
+Async Support
+-------------
+
+All methods have Async versions that are non-blocking so that you can use them on web applications for higher throughput, or GUI applications to prevent the UI from locking up while waiting on a response. Async versions are not available in the .NET 3.5 build.
```

Comment:
the readme looks great - I assume nothing special is required using the await methods? You can just continue coding like you were writing blocking code? 

Sorry, I'm an await noob. :)